src/org/aavso/tools/vstar/ui/dialog/plugin/manager/PluginManager.java

```diff
@@ -913,4 +936,22 @@ private List<byte[]> getBytesFromURL(URL url) throws IOException {
 
 		return byteList;
 	}
+
+//	private List<byte[]> getBytesFromURL(URL url) throws IOException {
```

Comment:
Feel free to just delete this and use this name for your new implementation Max.